documentation/docs/release-notes/3.3.0.md

```diff
@@ -0,0 +1,33 @@
+# Percona Monitoring and Management 3.2.0 
+
+| **Release date** | May 29th, 2025                                                                                  |
+| ----------------- | :---------------------------------------------------------------------------------------------- |
+| **Installation** | [Installing Percona Monitoring and Management](../quickstart/quickstart.md) |
+| **Upgrade**| [Migrate PMM 2 to PMM 3](../pmm-upgrade/migrating_from_pmm_2.md) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB. PMM empowers you to: 
+
+- monitor the health and performance of your database systems
+- identify patterns and trends in database behavior
+- diagnose and resolve issues faster with actionable insights
+- manage databases across on-premises, cloud, and hybrid environments
+
+## Release summary
+
+## Release highlights
+
+### TLS Skip Verify option for External Services
```

Comment:
Do you think it should be release highlight?

Reply:
removed older duplicate info